index.html

```diff
@@ -21,150 +20,188 @@
+                <li> <a href="https://www.linkedin.com/in/daniel-njoroge-3651a2a5/" target="_blank" rel="noopener"><img
+                            src="img/Linkedin.png" alt="linkedin"></a> </li>
+                <li> <a href="https://twitter.com/Icharian" target="_blank" rel="noopener"> <img src="img/Twitter.png"
+                            alt="twitter"></a></li>
+                <li> <a href="https://github.com/Dinicharia" target="_blank" rel="noopener"> <img src="img/github.png"
+                            alt="github"></a></li>
+                <li> <a href="www.angelink.com" target="_blank" rel="noopener"> <img src="img/angela.png"
+                            alt="angela"></a></li>
+                <li> <a href="www.medium.com" target="_blank" rel="noopener"> <img src="img/M.png" alt="m"></a></li>
+            </ul>
         </div>
     </section>
-    <div class="gridsection">
-        <section class="cardworks">
-            <div class="container">
-                <div class="workimage"><img src="img/Snapshoot.png" alt="snapshoot"></div>
-                <h1 class="greet">Tonic</h1>
-                <div class="frame2">
-                    <div class="client">CANOPY</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">Back End Dev</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">2015</div>
+    <section class="works">
+        <div class="work-container">
+            <div class="work">
+                <div class="img-container">
+                    <img src="img/Snapshoot.png" class="work-image" alt="work1">
                 </div>
-                <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <div class="tags">
-                    <p class="tag">html</p>
-                    <p class="tag">css</p>
-                    <p class="tag">javascript</p>
+                <div class="work-details">
+                    <h2 class="work-title">Tonic</h2>
+                    <div class="work-subtitle">
+                        <p class="company">Canopy</p>
+                        <img src="img/Counter.png" width="10" height="10" alt="counter">
+                        <p>Back End Dev</p>
+                        <img src="img/Counter.png" alt="counter">
+                        <p>2015</p>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <ul class="tags">
+                        <li class="tag">html</li>
+                        <li class="tag">css</li>
+                        <li class="tag">javascript</li>
+                    </ul>
+                    <a href="#" class="work-link">See Project</a>
```

Comment:
- [ ] Kindly add the styles for hover, active... to all the buttons on your page including the `See Project` and `Get in touch buttons, this will make the buttons on your page more interactive

|Your design|Figma design|
|-|-|
|![Screenshot_20220825_180800](https://user-images.githubusercontent.com/25227022/186729564-9d0c39d0-293a-43e6-b123-b5fc48e1ec3b.png)![Screenshot_20220825_180816](https://user-images.githubusercontent.com/25227022/186729572-a89b61bd-bf77-4a47-a7e8-73d008697d5b.png)![Screenshot_20220825_180838](https://user-images.githubusercontent.com/25227022/186729575-96920fc5-1434-41f7-a5a7-4667b2e8bdcb.png)|![Screenshot_20220825_181303](https://user-images.githubusercontent.com/25227022/186729592-5044457a-9937-4bfe-84d5-ae83c154be5e.png)|

Reply:
Currently, among the button interactions recommended by your chosen design, the enabled and hover states are implemented. However, you're yet to implement the _button-pressed_ interaction. Kindly do so as recommended by the previous reviewer. This helps to enhance user experience on your website.